spec/spec_helper.rb

```diff
@@ -1,19 +1,15 @@
 require 'simplecov'
 require 'simplecov-lcov'
 
+core_root = './lib/chess/core/'
 require 'byebug'
-require './lib/chess/chess'
-require './lib/chess/pieces/piece'
-require './lib/chess/pieces/king'
-require './lib/chess/pieces/queen'
-require './lib/chess/pieces/bishop'
-require './lib/chess/pieces/knight'
-require './lib/chess/pieces/rook'
-require './lib/chess/pieces/pawn'
-require './lib/chess/position'
-require './lib/chess/board/board'
-require './lib/chess/book'
-require './lib/chess/prompt'
+require "#{core_root}chess"
+require "#{core_root}cell"
+require "#{core_root}pieces/base_piece"
+require "#{core_root}pieces/bishop"
+require "#{core_root}pieces/knight"
+require "#{core_root}pieces/queen"
+require "#{core_root}pieces/pawn"
```

Comment:
For specs speed I'd suggest not requiring all the project files here. What is usually done is that you add `require` on each spec file as you need them